server/src/server.ts

```diff
@@ -800,32 +800,32 @@ connection.onTypeDefinition(async (typeDefinitionParams) => {
 })
 
 connection.onReferences(async (referencesParams) => {
-	const doc = documents.get(referencesParams.textDocument.uri)
-	if (!doc)
-		return null
 	const fileData = await getDocumentData(referencesParams.textDocument.uri)
 	if (!fileData)
 		return null
-	const callChain = findCallChain(doc, fileData, referencesParams.position, /*forAutocompletion*/false)
+	const doc = documents.get(fileData.uri)
+	if (!doc)
+		return null
+	const callChain = findCallChain(doc, fileData, referencesParams.position, false)
 	if (callChain.length === 0)
 		return null
-	const last = callChain[callChain.length - 1]
-	const foundTokens = last.tokens
 
 	let result: Location[] = []
+	for (const res of callChain[callChain.length - 1].tokens) {
+		console.log("[onReferences] Found token", res)
```

Comment:
res is object, does it print something readable or [object object]?

Reply:
Yep, it prints interactable json

Reply:
it works only in development mode, in result it will be just [object object] (

Reply:
sad :(